### PR TITLE
zerobin: update to v1.0.5

### PR DESCRIPTION
**Pull request by ju1m**
Update zerobin and dependencies which require an update for this.

###### Motivation for this change
Use latest zerobin.

###### Things done
- [X] Update zerobin to latest release: 20160108 => v1.0.5
- [X] Use `buildPythonApplication` instead of `buildPythonPackage`, and replace references in `services.zerobin` and documentation.
- [X] Update Beaker: 1.10.1 => 1.11.0
- [X] Update Paste: 3.4.1 => 3.4.3

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/zerobin/default.nix
Comment (ento) [inline pkgs/development/python-modules/zerobin/default.nix:12]: 🐛  Running the built executable failed in my environment:

```
/nix/store/dk6rznsq95h6m47nvcrjvxbg74qzj1mm-python3.8-zerobin-1.0.5/bin/zerobin
Traceback (most recent call last):
  File "/nix/store/dk6rznsq95h6m47nvcrjvxbg74qzj1mm-python3.8-zerobin-1.0.5/bin/.zerobin-wrapped", line 6, in <module>
    from zerobin.cli import main
  File "/nix/store/dk6rznsq95h6m47nvcrjvxbg74qzj1mm-python3.8-zerobin-1.0.5/lib/python3.8/site-packages/zerobin/__init__.py", line 3, in <module>
    __version__ = (Path(__file__).parent / "VERSION").read_text().strip()
  File "/nix/store/z65l1jqvxa58zzwwa3bvglb6asj4y8cv-python3-3.8.5/lib/python3.8/pathlib.py", line 1232, in read_text
    with self.open(mode='r', encoding=encoding, errors=errors) as f:
  File "/nix/store/z65l1jqvxa58zzwwa3bvglb6asj4y8cv-python3-3.8.5/lib/python3.8/pathlib.py", line 1218, in open
    return io.open(self, mode, buffering, encoding, errors, newline,
  File "/nix/store/z65l1jqvxa58zzwwa3bvglb6asj4y8cv-python3-3.8.5/lib/python3.8/pathlib.py", line 1074, in _opener 
    return self._accessor.open(self, flags, mode)
FileNotFoundError: [Errno 2] No such file or directory: '/nix/store/dk6rznsq95h6m47nvcrjvxbg74qzj1mm-python3.8-zerobin-1.0.5/lib/python3.8/site-packages/zerobin/VERSION'
```

Running the pydoit build script in the build phase seems to have fixed it:

<details><summary>diff</summary>


```diff
diff --git a/pkgs/development/python-modules/zerobin/default.nix b/pkgs/development/python-modules/zerobin/default.nix
index 6c80226bd68..03a95c75364 100644
--- a/pkgs/development/python-modules/zerobin/default.nix
+++ b/pkgs/development/python-modules/zerobin/default.nix
@@ -7,8 +7,11 @@
 , bottle
 , cherrypy
 , clize
+, doit
 , lockfile
 , paste
+, pyscss
+, nodePackages
 }:
 
 buildPythonPackage {
@@ -22,8 +25,13 @@ buildPythonPackage {
     sha256 = "sha256-JvgPWkycHJApHZJcvf3Oxm0fq625jr0FJ5/6IRAc3rU";
   };
 
+  nativeBuildInputs = [ doit pyscss nodePackages.uglify-js ];
   propagatedBuildInputs = [ appdirs beaker bleach bottle cherrypy clize lockfile paste ];
 
+  buildPhase = ''
+    doit build
+  '';
+
   # zerobin doesn't have any tests, but includes a copy of cherrypy which
   # can wrongly fail the check phase.
   doCheck = false;
```

</details>

**Review by ento — COMMENTED** [review]
(I'm following the PR template's suggestion to review open PRs to pay back/forward the work of reviewing, and this is my first time reviewing a nixpkgs PR. I'll defer to experienced reviewers regarding best practices followed in this repo.)

Reviewed points:
* Ensure that the commit text fits the guidelines.
* Built the package locally on Debian/x86_64.
* Executable runs on Debian/x86_64. (Failed with an error; commented with a diff that made it run)

File: pkgs/development/python-modules/beaker/default.nix
Comment (jonringer) [inline pkgs/development/python-modules/beaker/default.nix:29]:
```suggestion
    sha256 = "059sc7iar90lc2y9mppdis5ddfcxyirz03gmsfb0307f5dsa1dhj";
```
I would prefer not to have sri hashes, for supporting older versions of nix cli

```
$ nix to-base32 sha256-EragdCvugAGW0/UN8HP0nbnWio7t3pq8YBSkrOJhOhU=
059sc7iar90lc2y9mppdis5ddfcxyirz03gmsfb0307f5dsa1dhj
```

File: pkgs/development/python-modules/zerobin/default.nix
Comment (jonringer) [inline pkgs/development/python-modules/zerobin/default.nix:22]: please change to base32

File: pkgs/development/python-modules/zerobin/default.nix
Comment (ju1m) [reply pkgs/development/python-modules/zerobin/default.nix]: As put in the comment, I had to leave `doCheck = false`. but maybe you know a better way to avoid the error?

**Comment by ju1m** [comment]
@ento thanks a lot for you extensive reviewing, I wasn't expecting so much help :) I tried to incorporate all your suggestions in my last push.

File: pkgs/development/python-modules/zerobin/default.nix
Comment (ento) [reply pkgs/development/python-modules/zerobin/default.nix]: I found [one previous case](
https://github.com/NixOS/nixpkgs/pull/44571#issuecomment-410944462) where `doCheck = false` was actually encouraged if there are no tests to run, so I think we're good with leaving it in.

Regarding the error:

> nix_run_setup runserver: Received extra arguments: test

It looks like the root cause of this is the same as what's described in this SO answer:   https://stackoverflow.com/a/52550011

- `buildPythonApplication`'s `checkPhase` runs `setup.py test`
- `setup.py test` tries to find tests by importing modules
- [`zerobin/__main__.py` runs the main function](https://github.com/Tygs/0bin/blob/v1.0.5/zerobin/__main__.py#L5) upon being imported
- Because `test` is not a recognized command, [`runserver` is assumed to be the desired command](https://github.com/Tygs/0bin/blob/v1.0.5/zerobin/cli.py#L218-L219), but it still fails because the `runserver` function isn't expecting an argument like `test`

Changing `__main__.py` to wrap the `main()` call with a guard like described in the SO answer should do away with the error.

```python
if __name__ == "__main__":
    main()
```

This can be a PR against upstream - or a patch in this package, but if we're leaving `doCheck = false` in, then it may not make much sense to maintain a patch for something that's not strictly required.

File: doc/languages-frameworks/python.section.md
Comment (ento) [inline doc/languages-frameworks/python.section.md:1498]: It might be worth preserving the mention that NixOS services will also inherit overrides, if there's a Python package that's also used in a NixOS service and we can use that instead of scipy.

Looking at `git grep -E "ExecStart.*python.?Packages\." -- nixos/modules/` .. twisted can be a candidate.

```
nixos/modules/services/monitoring/graphite.nix:          ExecStart = "${pkgs.python3Packages.twisted}/bin/twistd ${carbonOpts name}";
```

Both graphite and twisted will probably be around for a while.

File: nixos/modules/services/networking/zerobin.nix
Comment (ento) [inline nixos/modules/services/networking/zerobin.nix:91]: Oh! If zerobin is considered a `networking/` service, then the package definition can be also under `pkgs/applications/networking/`.

File: pkgs/applications/misc/zerobin/default.nix
Comment (ento) [inline pkgs/applications/misc/zerobin/default.nix:17]: Another SRI hash has snuck in

```
$ nix to-base32 "sha256-JvgPWkycHJApHZJcvf3Oxm0fq625jr0FJ5/6IRAc3rU"
1dfy3h823ylz4w2vv3mrmnmiyvf6rvyvsp4j3llr074w9id0zy16
```

File: pkgs/development/python-modules/beaker/default.nix
Comment (ento) [inline pkgs/development/python-modules/beaker/default.nix:69]: Hmm, it seems like adding `glibcLocales` to `checkInputs` is a bit more customary in the code base:

```
$ git grep checkInputs | grep glibcLocale | wc -l
66
$ git grep LOCALE_ARCHIVE | wc -l              
45
```

Also, the "This PR does not cleanly list package outputs after merging" test failure is likely coming from the `${glibcLocales}` interpolation, as the error message in the logs says ["cannot coerce null to a string"](https://gist.github.com/GrahamcOfBorg/404709bde9555f52dc25cfac78bf3ace) and `glibcLocales` is defined as `null` when the host platform is not on glibc.

https://github.com/NixOS/nixpkgs/blob/27f084d817d6486ab0c2c23aff4305404646c2fa/pkgs/top-level/all-packages.nix#L12655

(sorry, didn't check this in my first review)

File: pkgs/applications/misc/zerobin/default.nix
Comment (ento) [inline pkgs/applications/misc/zerobin/default.nix:23]: 📝 In my previous review, I [raised one concern](https://github.com/NixOS/nixpkgs/pull/98734#discussion_r495502385) about how zerobin [pins its dependencies to exact versions](https://github.com/Tygs/0bin/blob/v1.0.5/setup.cfg#L15-L22). Since then, I looked around at how strict dependencies are dealt with; it seems like using a patch file to relax those dependencies would be a workaround when it comes to it. 

- Looked for imports done: `git grep buildPythonApplication -- pkgs | cut -d: -f1 | grep -v all-packages | grep -v python-packages.nix | xargs grep -E "import.*\.nix"`
  [nixpart v0.4](https://github.com/NixOS/nixpkgs/tree/27f084d817d6486ab0c2c23aff4305404646c2fa/pkgs/tools/filesystems/nixpart/0.4) seems to be about the only package that uses its own dependency definitions instead of `pythonPackages` / `python3Packages` etc.
- Came across [one package](https://github.com/NixOS/nixpkgs/blob/27f084d817d6486ab0c2c23aff4305404646c2fa/pkgs/servers/home-assistant/relax-dependencies.patch) that uses a patch to relax some dependencies, [and another](https://github.com/NixOS/nixpkgs/blob/27f084d817d6486ab0c2c23aff4305404646c2fa/pkgs/development/python-modules/hwi/relax-deps.patch)

In short, I think this is fine and nothing to do here further.

File: doc/languages-frameworks/python.section.md
Comment (ju1m) [reply doc/languages-frameworks/python.section.md:1498]: @ento I've replaced `scipy` by `twisted` as you suggested.

File: nixos/modules/services/networking/zerobin.nix
Comment (ju1m) [reply nixos/modules/services/networking/zerobin.nix:91]: @ento I've moved to `zerobin` to `pkgs/applications/networking/` as you suggested

File: pkgs/development/python-modules/beaker/default.nix
Comment (ju1m) [reply pkgs/development/python-modules/beaker/default.nix]: @ento I've added `glibcLocales` into `checkInputs` as you suggested, thanks!

File: pkgs/development/python-modules/zerobin/default.nix
Comment (ju1m) [reply pkgs/development/python-modules/zerobin/default.nix]: @ento, ok I'm leaving `doCheck = false`, thanks!

File: doc/languages-frameworks/python.section.md
Comment (ento) [inline doc/languages-frameworks/python.section.md:1499]: Feel free to take or leave this suggestion: the reader might need to connect the dots that `graphite` uses `twisted` - so maybe

```suggestion
`pythonPackages.twisted` is now globally overridden. All packages and also the
`graphite` NixOS service, which uses `twisted`, use the new definition. Note that `python-super` refers
```

or 

```suggestion
`pythonPackages.twisted` is now globally overridden. All packages and also all
NixOS services that reference `twisted` use the new definition. Note that `python-super` refers
```

File: pkgs/applications/networking/zerobin/default.nix
Comment (ento) [inline pkgs/applications/networking/zerobin/default.nix:41]: 🤔 My understanding is that all Nix packages are expected to have a maintainer, but the zerobin package was already without one. I'm not sure how the policy is applied in everyday practice, so I'll leave it to other reviewers.

**Review by ento — APPROVED** [review]
@ju1m Thanks for the updates!

@jonringer @domenkozar @FRidh Could one of you pick up the rest of the review process? As an apprentice reviewer, here's the farthest I'm equipped to contribute :)

Reviewed points so far:
* Ensure that the commit text fits the guidelines.
* Built the package locally on Debian/x86_64.
* Executable runs on Debian/x86_64.

File: doc/languages-frameworks/python.section.md
Comment (ju1m) [reply doc/languages-frameworks/python.section.md]: Thanks, I've merged your second suggestion.

File: pkgs/applications/networking/zerobin/default.nix
Comment (ju1m) [reply pkgs/applications/networking/zerobin/default.nix:41]: At this point I haven't used `zerobin` yet, so I'm not sure if I will care enough about it to add myself as a maintainer. For me this PR was just to give a helping hand to a friend testing `zerobin`.

AFAIK this is what it means to be a maintainer:
```
a maintainer will do a best effort to help resolving any issues
with the package explicitly brought to their attention,
and believes they can be useful in that. 
```
Source: https://logs.nix.samueldr.com/nixos-dev/2020-05-10#1589147927-1589147984;
But I have not searched the RFCs to see if there is something more.

**Comment by ju1m** [comment]
Removed the commit updating `pythonPackages.paste` since it has been done in 77fc15da7c757e54ba0098b5de389785451dbe57

File: pkgs/top-level/all-packages.nix
Comment (doronbehar) [inline pkgs/top-level/all-packages.nix:25869]: There's this rule of thumb that recommends using in this case:

```nix
  zerobin = callPackage ../.... { };
```

And use `python3Packages.buildPythonApplication` inside the expression.

**Review by doronbehar — COMMENTED** [review]
Other then that and the merge conflicts, LGTM.

File: pkgs/top-level/all-packages.nix
Comment (ju1m) [reply pkgs/top-level/all-packages.nix]: @doronbehar I've changed as you suggested, and rebased after changes from a6ce00c50c36624fec06b2b756a766d4d0f4a888

File: pkgs/applications/networking/zerobin/default.nix
Comment (doronbehar) [inline pkgs/applications/networking/zerobin/default.nix:10]: Here there should be `python3.pkgs.buildPythonApplications`.

File: pkgs/development/python-modules/zerobin/default.nix
Comment (doronbehar) [inline pkgs/development/python-modules/zerobin/default.nix:8]: And this file should be removed.

**Review by doronbehar — COMMENTED** [review]
Eval fails currently due to these.

File: pkgs/applications/networking/zerobin/default.nix
Comment (ju1m) [reply pkgs/applications/networking/zerobin/default.nix]: Oops, sorry I should have tested the build before uploading. Because the replacing of `python3Packages.callPackage` by `callPackage` in last change, `pythonOlder` needed to be imported from `python3Packages`.
Note that instead of `python3.pkgs` I'm using the `python3Packages` alias, just because it's what is suggested in https://nixos.org/manual/nixpkgs/stable/#python-library-packages-in-nixpkgs

**Comment by ju1m** [comment]
Rebased on `master`, I had to disable `pyScss` checkPhase and patch `setup.cfg` to relax versions of some dependencies.

File: pkgs/applications/networking/zerobin/default.nix
Comment (doronbehar) [inline pkgs/applications/networking/zerobin/default.nix:20]: I would not import `doit` from `all-packages.nix` and use `doit` right after `with python3Packages`, although the result is similar. And to make that even clearer, I would put here:

```suggestion
  nativeBuildInputs = [
    python3Packages.doit
    python3Packages.pycss
    nodePackages.uglify-js
  ];
```

File: pkgs/applications/networking/zerobin/default.nix
Comment (doronbehar) [inline pkgs/applications/networking/zerobin/default.nix:5]: And remove this line:

```suggestion
```

File: pkgs/applications/networking/zerobin/default.nix
Comment (doronbehar) [inline pkgs/applications/networking/zerobin/default.nix:42]: Maintainers? Platforms?

File: pkgs/development/python-modules/zerobin/default.nix
Comment (doronbehar) [inline pkgs/development/python-modules/zerobin/default.nix:1]: So is this file - `python-modules/zerobin/default.nix` being used anywhere?

File: doc/languages-frameworks/python.section.md
Comment (doronbehar) [inline doc/languages-frameworks/python.section.md:1504]: Just a suggestion you don't have to take it :)

```suggestion
(such as `services.buildbot-worker`) now use the new definition.
```

File: pkgs/applications/networking/zerobin/default.nix
Comment (doronbehar) [inline pkgs/applications/networking/zerobin/default.nix:32]: `runHook preBuild` and `runHook postBuild`.

File: pkgs/development/python-modules/zerobin/default.nix
Comment (ju1m) [reply pkgs/development/python-modules/zerobin/default.nix]: Oh, right, that's the old one, I've removed it, thanks.

File: pkgs/applications/networking/zerobin/default.nix
Comment (ju1m) [reply pkgs/applications/networking/zerobin/default.nix:53]: I'm adding myself as maintainer in becase I might help others updating/fixing `zerobin`'s packaging in the future, but I'm not using it, so I won't be updating it by my own.

File: pkgs/applications/networking/zerobin/default.nix
Comment (ju1m) [reply pkgs/applications/networking/zerobin/default.nix]: Done, thanks.

File: pkgs/applications/networking/zerobin/default.nix
Comment (ju1m) [reply pkgs/applications/networking/zerobin/default.nix]: Done, thanks.

File: pkgs/applications/networking/zerobin/default.nix
Comment (ju1m) [reply pkgs/applications/networking/zerobin/default.nix:43]: Is it a suggestion?

Comment (doronbehar) [reply pkgs/applications/networking/zerobin/default.nix:43]: > Is it a suggestion?

Yes please, see: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md .

Comment (ju1m) [reply pkgs/applications/networking/zerobin/default.nix:43]: Done, thanks!

**Comment by doronbehar** [comment]
Result of `nixpkgs-review pr 98734` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages built:</summary>
  <ul>
    <li>python38Packages.beaker</li>
    <li>python38Packages.pyramid_beaker</li>
    <li>python38Packages.pyscss</li>
    <li>python39Packages.beaker</li>
    <li>python39Packages.pyramid_beaker</li>
    <li>python39Packages.pyscss</li>
    <li>zerobin</li>
  </ul>
</details>

**Comment by doronbehar** [comment]
```
(ins)[nix-shell:~/.cache/nixpkgs-review/pr-98734]$ ./results/zerobin/bin/zerobin --help
Traceback (most recent call last):
  File "/nix/store/44745hzps8f24gch6jx461kilbwk5jc6-zerobin-1.0.5/bin/.zerobin-wrapped", line 6, in <module>
    from zerobin.cli import main
  File "/nix/store/44745hzps8f24gch6jx461kilbwk5jc6-zerobin-1.0.5/lib/python3.8/site-packages/zerobin/cli.py", line 22, in <module>
    from zerobin.routes import get_app
  File "/nix/store/44745hzps8f24gch6jx461kilbwk5jc6-zerobin-1.0.5/lib/python3.8/site-packages/zerobin/routes.py", line 23, in <module>
    from beaker.middleware import SessionMiddleware
  File "/nix/store/jrcqj6iap1565drvcr0b8b3r2fpczfz5-python3.9-Beaker-1.11.0/lib/python3.9/site-packages/beaker/middleware.py", line 11, in <module>
    from beaker.cache import CacheManager
  File "/nix/store/jrcqj6iap1565drvcr0b8b3r2fpczfz5-python3.9-Beaker-1.11.0/lib/python3.9/site-packages/beaker/cache.py", line 15, in <module>
    from beaker.crypto.util import sha1
  File "/nix/store/jrcqj6iap1565drvcr0b8b3r2fpczfz5-python3.9-Beaker-1.11.0/lib/python3.9/site-packages/beaker/crypto/__init__.py", line 5, in <module>
    from beaker.crypto.util import hmac, sha1, hmac_sha1, md5
  File "/nix/store/jrcqj6iap1565drvcr0b8b3r2fpczfz5-python3.9-Beaker-1.11.0/lib/python3.9/site-packages/beaker/crypto/util.py", line 5, in <module>
    from Crypto.Hash import HMAC as hmac, SHA as hmac_sha1
  File "/nix/store/igpkzdypdjfzzj8iz960pcgik51b6mms-python3.9-pycryptodome-3.9.9/lib/python3.9/site-packages/Crypto/Hash/HMAC.py", line 38, in <module>
    from Crypto.Hash import MD5
  File "/nix/store/igpkzdypdjfzzj8iz960pcgik51b6mms-python3.9-pycryptodome-3.9.9/lib/python3.9/site-packages/Crypto/Hash/MD5.py", line 29, in <module>
    _raw_md5_lib = load_pycryptodome_raw_lib("Crypto.Hash._MD5",
  File "/nix/store/igpkzdypdjfzzj8iz960pcgik51b6mms-python3.9-pycryptodome-3.9.9/lib/python3.9/site-packages/Crypto/Util/_raw_api.py", line 308, in load_pycryptodome_raw_lib
    raise OSError("Cannot load native module '%s': %s" % (name, ", ".join(attempts)))
OSError: Cannot load native module 'Crypto.Hash._MD5': Trying '_MD5.cpython-38-x86_64-linux-gnu.so': cannot load library '/nix/store/igpkzdypdjfzzj8iz960pcgik51b6mms-python3.9-pycryptodome-3.9.9/lib/python3.9/site-packages/Crypto/Util/../Hash/_MD5.cpython-38-x86_64-linux-gnu.so': /nix/store/igpkzdypdjfzzj8iz960pcgik51b6mms-python3.9-pycryptodome-3.9.9/lib/python3.9/site-packages/Crypto/Util/../Hash/_MD5.cpython-38-x86_64-linux-gnu.so: cannot open shared object file: No such file or directory.  Additionally, ctypes.util.find_library() did not manage to locate a library called '/nix/store/igpkzdypdjfzzj8iz960pcgik51b6mms-python3.9-pycryptodome-3.9.9/lib/python3.9/site-packages/Crypto/Util/../Hash/_MD5.cpython-38-x86_64-linux-gnu.so', Trying '_MD5.abi3.so': cannot load library '/nix/store/igpkzdypdjfzzj8iz960pcgik51b6mms-python3.9-pycryptodome-3.9.9/lib/python3.9/site-packages/Crypto/Util/../Hash/_MD5.abi3.so': /nix/store/igpkzdypdjfzzj8iz960pcgik51b6mms-python3.9-pycryptodome-3.9.9/lib/python3.9/site-packages/Crypto/Util/../Hash/_MD5.abi3.so: cannot open shared object file: No such file or directory.  Additionally, ctypes.util.find_library() did not manage to locate a library called '/nix/store/igpkzdypdjfzzj8iz960pcgik51b6mms-python3.9-pycryptodome-3.9.9/lib/python3.9/site-packages/Crypto/Util/../Hash/_MD5.abi3.so', Trying '_MD5.so': cannot load library '/nix/store/igpkzdypdjfzzj8iz960pcgik51b6mms-python3.9-pycryptodome-3.9.9/lib/python3.9/site-packages/Crypto/Util/../Hash/_MD5.so': /nix/store/igpkzdypdjfzzj8iz960pcgik51b6mms-python3.9-pycryptodome-3.9.9/lib/python3.9/site-packages/Crypto/Util/../Hash/_MD5.so: cannot open shared object file: No such file or directory.  Additionally, ctypes.util.find_library() did not manage to locate a library called '/nix/store/igpkzdypdjfzzj8iz960pcgik51b6mms-python3.9-pycryptodome-3.9.9/lib/python3.9/site-packages/Crypto/Util/../Hash/_MD5.so'

```

**Comment by ju1m** [comment]
@doronbehar , hmm, I cannot reproduce, it works for me. For you it looks like `beaker` is importing `pycryptodome` but it then fails to load `pycryptodome-3.9.9/lib/python3.9/site-packages/Crypto/Util/../Hash/_MD5.cpython-38-x86_64-linux-gnu.so`.
Maybe `pycryptodome` should be added to the `propagatedBuildInputs` of `beaker`, I don't know.

**Comment by ju1m** [comment]
I can reproduce when using `nixpkgs-review pr 98734`.

**Comment by doronbehar** [comment]
> @doronbehar, I can reproduce when using `nixpkgs-review pr 98734`.

Then it means you need to rebase this on top of latest master and fix it then.

**Comment by doronbehar** [comment]
@ju1m I noticed now that the stacktrace I quoted previously had mixed versions of Python used together, and that's probably why it failed. Now I rebased your branch and checked everything once more and zerobin runs fine. Merging.

**Pull request merged**
